src/Components/Header/SearchBarAndIcon/Icons.js

```diff
@@ -4,8 +4,8 @@ import styles from "./Icons.module.css";
 export default function Icons() {
     return (
      <div className={styles.Icons}>
-      <FavoriteBorderOutlinedIcon/>
-      <ShoppingCartOutlinedIcon/>
+      <a href='#'><FavoriteBorderOutlinedIcon/></a>
+      <a href="#"><ShoppingCartOutlinedIcon/></a>
```

Comment:
Why not have an array that contains FavoriteBorderOutlinedIcon and ShoppingCartOutlinedIcon, and then loop over it to return an anchor with the element from the array being the content of the anchor tag?